### ISSUE TITLE
Zurich forbidden issue

### ISSUE THREAD

**Issue by sushilshinde** [topcoder-platform/community-app#3364]
Private communities are being forbidden because of the fix for 

https://github.com/topcoder-platform/community-app/issues/3331

Fix commit - https://github.com/topcoder-platform/community-app/commit/43645476acb63544fc6129db0e96d58202197a30

use `zurich-fixes` community-app branch

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30103832 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by sushilshinde** [comment]
Not reproducible anymore.

@mtwomey @nithyaasworld @r0hit-gupta

**Comment by nithyaasworld** [comment]
This was the way I was able to reproduce:

1. Open new incognito (important step)
2. Go to zurich.topcoder.com (forbidden message is shown, which is expected at this point)
3. Then go to topcoder.com and login with topcoder id
4. Then refresh the page opened in step:2 and check that forbidden message is still shown

On step 4, I am actually accessing the URL zurich.topcoder.com**/forbidden** by refreshing the page. So I ended up getting the forbidden message again 🙂 

I think its the usual behaviour for any user to refresh the page when something is not loading up! But in this case, user needs to remove the **/forbidden** from the URL and then refresh.
